pkgs/development/interpreters/python/tests/test_nix_pythonprefix/default.nix

```diff
@@ -0,0 +1,25 @@
+{ interpreter, pkgs }:
```

Comment:
more specific parameters than `pkgs`?

Reply:
Agreed, also switched to using `callPackage`.